src/components/Auth/LoginForm.js

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { Box, Button, TextField } from '@material-ui/core/';
+import axios from 'axios';
+
+const LoginForm = ({ toggleActiveForm }) => {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+
+  const handleLogin = e => {
+    e.preventDefault();
+    const data = {
+      username,
+      password,
+    };
+    axios
+      .post('https://localhost:8000/auth/obtain_token/', data)
+      .then(res => {
+        localStorage.setItem('userData', res.data);
```

Comment:
Ah yeah, I was thinking of using `js-cookie`! Added this to the original post yesterday:

```
[ ] New ticket: consider using js-cookie instead of using localStorage
```
`res.data` returns the following:

```
{
    "username": "Lester.Frami66",
    "token": "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VybmFtZSI6Ikxlc3Rlci5GcmFtaTY2IiwiaWF0IjoxNTgzNzAyNTU3LCJleHAiOjE1ODM3MDYxNTcsInVzZXJfaWQiOjksIm9yaWdfaWF0IjoxNTgzNzAyNTU3fQ.SM-rIUwfGM_00nUIE3qmTFUJb6Y2TQx4VBNmo0CQh0E",
    "first_name": "Alexis",
    "last_name": "Goldner",
    "email": "Thalia95@hotmail.com"
}
```